### PR TITLE
Improve

### PR DESCRIPTION
**Pull request by avsm**

### CODE REVIEW

File: _tags [hunk 1]
```diff
@@ -1,4 +1,4 @@
-true : bin_annot, safe_string, keep_locs
+true : bin_annot, safe_string, keep_locs, debug
```
Comment (hannesm) [inline _tags]: added automatically by topkg, see https://github.com/dbuenzli/topkg/issues/54

File: doc/dev.odocl [hunk 2]
```diff
@@ -0,0 +1 @@
+Parse_argv
```
Comment (hannesm) [inline doc/dev.odocl:1]: this file doesn't seem to be needed

Comment (avsm) [reply doc/dev.odocl:1]: its used if you `topkg doc --dev`

File: _tags (hunk 1 above)
Comment (avsm) [reply _tags]: fixed in 6a00058baa6f6c9c74376a533449537402bd793c

**Pull request merged**